app/models/mpc_offender.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+class MpcOffender
+  delegate :tier, to: :@probation_record
+
+  delegate :get_image,
+           :convicted?, :recalled?, :immigration_case?, :indeterminate_sentence?,
+           :sentenced?, :over_18?, :describe_sentence, :civil_sentence?,
+           :sentence_start_date, :conditional_release_date, :automatic_release_date, :parole_eligibility_date,
+           :tariff_date, :post_recall_release_date, :licence_expiry_date,
+           :home_detention_curfew_actual_date, :home_detention_curfew_eligibility_date,
+           :prison_arrival_date, :earliest_release_date, :latest_temp_movement_date,
+           :date_of_birth, :main_offence, :awaiting_allocation_for, :cell_location,
+           :category_label, :complexity_level, :category_code, :category_active_since,
+           :first_name, :last_name, :full_name_ordered, :full_name,
+           :inside_omic_policy?, :offender_no, :prison_id, to: :@prison_record
+
+  delegate :crn, :case_allocation, :parole_review_date, :manual_entry?, :nps_case?,
+           :team_name, :ldu_name, :ldu_email_address,
+           :victim_liaison_officers, :early_allocations,
+           :mappa_level, :welsh_offender, to: :probation_record
+
+  attr_reader :probation_record
+
+  def initialize(offender, api_offender)
+    @offender = offender
+    @prison_record = api_offender
+    @probation_record = offender.case_information
+  end
```

Comment:
We also talked about passing the offender's current `Prison` into the initialiser, and making it available at `attr_reader @prison`.

Would be lovely to have that...

Reply:
Also:

This could just be personal preference, but I quite like the idea of accepting `probation_record` as its own initialiser argument. So the OffenderService knows how to find the probation record, rather than the initialiser knowing that it exists 'inside' the Offender record.

One subtle advantage of this is that it should simplify the way we structure tests.

### Example

For example: `spec/helpers/offender_helper_spec.rb`

Where we currently have:

```ruby
let(:offender) {
  build(:mpc_offender, prison_record: api_offender, offender: build(:case_information).offender)
}
```

We could change this to:

```ruby
let(:offender) {
    build(:mpc_offender,
        prison_record: api_offender,
        probation_record: build(:case_information),
        offender: build(:offender))
}
```

So we don't have to build a `CaseInformation` just to extract the `Offender` record (only for the initialiser to extract the `CaseInformation` from the `Offender` again...)

The `CaseInformation` records could almost be cut loose from the main `Offender` data model... and exist 'on the side' of our internal data model, only being associated with the `LocalDeliveryUnit`.